### PR TITLE
Documentation changes - #7760

### LINKED ISSUE #7760: Documentation Issues - 2020a
Issue overview
--------------
A collection of small documentation issues. See checkboxed comments below.

### PR DESCRIPTION
**Pull request by jmythms**
Pull request overview
---------------------
 - Fixes #7760
 - A few fixes to correct documentation issues

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [x] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix

### CODE REVIEW

**Comment by mjwitte** [comment]
@jmythms Is this ready for review?

**Comment by jmythms** [comment]
@mjwitte Yes, It would be great if you could review it for me. Thank you.

File: doc/input-output-reference/src/overview/group-airflow-network.tex [hunk 1]
```diff
@@ -1,4 +1,4 @@
-\section{Group -- Airflow Network}\label{group-airflow-network}
+\section{Group -- Natural Ventilation and Duct Leakage}\label{group-natural-ventilation-and-duct-leakage}
```
Comment (mjwitte) [inline doc/input-output-reference/src/overview/group-airflow-network.tex]: Per @jasondegraw 's https://github.com/NREL/EnergyPlus/issues/7760#issuecomment-585356366 we should revert this change and update the IDD post v9.4.

File: doc/input-output-reference/src/overview/group-water-heaters.tex [hunk 2]
```diff
@@ -1,4 +1,4 @@
-\section{Group -- Water Heaters}\label{group-water-heaters}
+\section{Group -- Water Heaters and Thermal Storage}\label{group-water-heaters}
```
Comment (mjwitte) [inline doc/input-output-reference/src/overview/group-water-heaters.tex:1]: Moving this is OK.

File: doc/input-output-reference/src/overview/group-humidifiers-and-dehumidifiers.tex [hunk 3]
```diff
@@ -1,4 +1,288 @@
-\section{Group Desiccant Dehumidifiers}\label{group-desiccant-dehumidifiers}
+\section{Group -- Humidifiers and Dehumidifiers}\label{group-humidifiers-and-dehumidifiers}
```
Comment (mjwitte) [inline doc/input-output-reference/src/overview/group-humidifiers-and-dehumidifiers.tex:1]: Combining these two groups is Ok.

File: doc/input-output-reference/src/overview/group-air-distribution-equipment.tex [hunk 4]
```diff
@@ -1,4 +1,4 @@
-\section{Group -- Air Distribution Equipment}\label{group-air-distribution-equipment}
+\section{Group -- Zone HVAC Air Loop Terminal Units}\label{group-zone-HVAC-air-loop-terminal-units}
```
Comment (mjwitte) [inline doc/input-output-reference/src/overview/group-air-distribution-equipment.tex:1]: I'm not a big fan of this group name - but there's also a group call "Air Distribution" which includes the main air loop components, so this name helps distinguish the terminal units.

File: doc/engineering-reference/src/loop-equipment-sizing-and-other-design-data/component-sizing.tex [hunk 5]
```diff
@@ -649,6 +649,52 @@ \subsubsection{Rated Sensible Cooling Capacity}\label{rated-sensible-cooling-cap
...
+CPC2 = user input for Cooling Power Consumption Coefficient 2
+
+CPC3 = user input for Cooling Power Consumption Coefficient 3
+
+CPC4 = user input for Cooling Power Consumption Coefficient 4
+
+CPC5 = user input for Cooling Power Consumption Coefficient 5
+
+
+The 4\(^{th}\) and 5\(^{th}\) coefficient (CPC4 and CPC5) used in the above equation are multipliers for the load-side and source-side flow ratios, respectively. For sizing, these ratios are assumed to be 1.
```
Comment (mjwitte) [inline doc/engineering-reference/src/loop-equipment-sizing-and-other-design-data/component-sizing.tex]: I see you've followed the pattern of the other two sets of coefficients above, but all of these should include the VLratio and VSratio terms in the equations (even if the term is assumed to be 1 for sizing). These same equations are used during the simulation if I understand correctly.

File: doc/engineering-reference/src/loop-equipment-sizing-and-other-design-data/component-sizing.tex [hunk 6]
```diff
@@ -716,9 +762,102 @@ \subsubsection{Rated Water Flow Rate}\label{rated-water-flow-rate-2}
...
 The calculation is identical to that done for \emph{Coil:Cooling:Water} , which is the coil design load divided by the \emph{Loop Design Temperature Difference} user input from \emph{Sizing:Plant.} For sizing the plant loop serving this coil, only one half of this flow rate is used since both the cooling and heating coil will save a flow rate but only one of these coils will operate at a time.
 
+%\subsubsection{Rated Total Heating Capacity}\label{rated-total-heating-capacity}
+
+%The rated total heating capacity is set equal to the rated total cooling capacity.
+
 \subsubsection{Rated Total Heating Capacity}\label{rated-total-heating-capacity}
 
-The rated total heating capacity is set equal to the rated total cooling capacity.
+The following calculations are performed to determine the rated total heating capacity.
```
Comment (mjwitte) [inline doc/engineering-reference/src/loop-equipment-sizing-and-other-design-data/component-sizing.tex]: Hmm, are you sure it's not set equal? Looks like the code sets [`RatedCapHeatDes = DXCoolCap;` here](https://github.com/NREL/EnergyPlus/blob/b1c7a5b4634ed662ef93c23b195ba084a09df3c6/src/EnergyPlus/WaterToAirHeatPumpSimple.cc#L1894). 
Actually, all of the coefficients are already explained in the Engineering Reference section titled "Single Speed Equation-Fit Model". Maybe not real obvious, but A1-F5 are noted after the equations and then identified in the following section "Coefficient estimation procedure". So the best remedy might be to add some text in the IO Reference pointing to these sections in the Engineering Reference.

**Review by mjwitte — COMMENTED** [review]
This is ok, but the entire IO Ref could use an outline restructuring independent of the IDD order. There are sections (like EMS) that are plopped in the middle of other related groups. Not suggesting we change anything more here. Most of these changes are OK for now.

Same for the Engineering Reference (6 identical chapter headings of "Simulation Models - Encyclopedic Reference"?).

Regarding @jasondegraw 's question about groups in epJSON. The \group names are not currently preserved in the schema, but the objects do appear to be in the same order as the IDD. How that order plays out in tools for editing epJSON remains to be seen (some grouping as a navigational aid would be helpful IMHO).

There are some items checked off in #7760 that are not fixed here (I unchecked them):
https://github.com/NREL/EnergyPlus/issues/7760#issuecomment-594215711 
https://github.com/NREL/EnergyPlus/issues/7760#issuecomment-595318925

File: doc/engineering-reference/src/loop-equipment-sizing-and-other-design-data/component-sizing.tex (hunk 6 above)
Comment (jmythms) [reply doc/engineering-reference/src/loop-equipment-sizing-and-other-design-data/component-sizing.tex]: You are right, my mistake. I got confused by the heating mode simulation code [here](https://github.com/NREL/EnergyPlus/blob/b1c7a5b4634ed662ef93c23b195ba084a09df3c6/src/EnergyPlus/WaterToAirHeatPumpSimple.cc#L2595).
Thanks, I'll make the changes in the IO reference.

**Comment by jasondegraw** [comment]
@mjwitte Ordering within schemas is a tricky thing, so if there's going to be some grouping going forward we probably should discuss that sooner rather than later. It might be worth a separate issue.

**Review by mjwitte — APPROVED** [review]
@jmythms Thanks for addressing comments. This looks good and will merge after CI completes its rounds.

**Comment by jmythms** [comment]
> @jmythms Thanks for addressing comments. This looks good and will merge after CI completes its rounds.

Thank you.
